fabric/src/main/resources/fabric.mod.json

```
@@ -28,7 +28,7 @@
   "depends": {
     "fabricloader": ">=0.16.14",
     "fabric": "*",
-    "minecraft": ">=1.21.6",
+    "minecraft": ">1.21.8",
```

```suggestion
    "minecraft": ">=1.21.9",
```
Prefer keeping it like this just because I think it reads a bit more clearly